openapi.yaml

```
@@ -1,7 +1,7 @@
 openapi: 3.0.0
 info:
   title: Unstoppable Partner API
-  description: 'The Partner API requires you to be authorized to access them. To access the Partner API and integrate it into your application, check our [Set up Partner API Access](https://docs.unstoppabledomains.com/partner/) guide.'
+  description: 'The Partner API requires you to be an authorized seller. To register as a partner and integrate our APIs into your applications, check our [Register as a Partner](https://docs.unstoppabledomains.com/partner/) guide.'
```

Let's use "partner" here
```suggestion
  description: 'The Partner API requires you to be an authorized partner. To register as a partner and integrate our APIs into your applications, check our [Register as a Partner](https://docs.unstoppabledomains.com/partner/) guide.'
```